pkg/apis/core/v1alpha1/types.go

```diff
@@ -141,12 +143,25 @@ type DataVolumeList struct {
 	Items []DataVolume `json:"items"`
 }
 
+// DataVolumeCondition represents the state of a data volume condition.
+type DataVolumeCondition struct {
```

Comment:
The timestamp fields on most conditions are missing `lastTransitionTime` and `lastProbeTime`/`LastHeartbeatTime`

Reply:
I purposely didn't add transition/lastprobe/heartbeart/other time. After doing some research I noticed basically the only conditions that used that was the conditions we put in for HCO. The time didn't add much/any value so why bother adding it?

Reply:
I think there is value.  At least in transition time.  And I've never encountered conditions without timestamps.  Here are some places that do:

https://github.com/kubevirt/kubevirt/blob/master/staging/src/kubevirt.io/client-go/api/v1/types.go#L266-L284

https://github.com/kubevirt/kubevirt/blob/master/staging/src/kubevirt.io/client-go/api/v1/types.go#L664-L671

https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L448-L460

https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L2250-L2262

https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L4009-L4021

Reply:
Okay, I have added transition time.